docs/design/preload.md

```diff
@@ -48,7 +48,7 @@ The MediaSourceEngine manages the MediaSource and the SourceBuffers.
 ## Design Overview
 
 The design will address two major requirements:
-1. Enable preloading a manifest in Player before attaching to a video element
+1. Enable preloading a manifest in Player before attaching it to a video element
```

Comment:
This change is wrong.  The Player is attaching (implied: itself) to the video element.  Please revert.